SketchUp/Sketchup/AttributeDictionary.rb

```diff
@@ -271,7 +271,7 @@ def size
   #   attrdict = model.attribute_dictionaries['test_dict']
   #   values = attrdict.values
   #
-  # @return keys - an array of keys within the attribute dictionary
+  # @return [Array<Object, nil>] an array of values within the attribute dictionary
```

Comment:
This can probably be just be `Array<Object>` since Ruby consider `NilClass` to inherit from `Object`.

```
nil.class.ancestors
# => [NilClass, Object, Kernel, BasicObject]
```